app/src/main/java/edu/akelael/comics/model/Server.java

```diff
@@ -0,0 +1,16 @@
+package edu.akelael.comics.model;
+
+import java.util.Map;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.QueryMap;
+
+public interface Server {
+    String baseUrl = "http://gateway.marvel.com/v1/public/";
+    int ID_AMAZING_SPIDERMAN = 1010733;
```

Comment:
Yes... but for interfaces isn't it redundant? But clean... Mmm I don't know what is the best decision. I think that with the modifiers is cleaner.